src/main/java/com/salesforce/storm/spout/sideline/handler/SidelineSpoutHandler.java

```diff
@@ -344,15 +339,18 @@ public void onSpoutClose(final DynamicSpout spout) {
      * @param sidelineRequest sideline request.
      * @return true it does, false it does not.
      */
-    public boolean isSidelineStarted(SidelineRequest sidelineRequest) {
+    @Override
+    public boolean isStarted(SidelineRequest sidelineRequest) {
         return fireHoseSpout.getFilterChain().hasStep(sidelineRequest.id);
     }
 
     /**
-     * Starts a sideline request.
-     * @param sidelineRequest representation of the request that is being started.
+     * Start processing on the firehose for a future sideline.
+     *
+     * @param sidelineRequest sideline request.
      */
-    public void startSidelining(SidelineRequest sidelineRequest) {
+    @Override
+    public void start(SidelineRequest sidelineRequest) {
```

Comment:
does there need to be any validation around, what if you attempt to START a sideline thats already in RESUME, etc..  Basically validation that this state machine can't run backwards.